Iterators.md

```diff
@@ -0,0 +1,89 @@
+# Iterators
+
+This section of tutorial will be focused on iterators. If you are unfamiliar with iterators in Python, you might find it helpful to read through the Python tutorial on iterators and generators, which you can find [here](https://docs.Python.org/3/tutorial/classes.html#iterators). This tutorial also references the Python itertools module, the documentation for which can be found [here](https://docs.Python.org/2/library/itertools.html).
+
+Python's iterator functionality can be complex and a bit difficult to use. A good chunk of it comes from itertools, which is a module that implements several iterator building blocks inspired by functional programming languages from Haskell and APL. Coconut simplifies many of these features and treats them as built-ins rather than add-ons., making using iterators easier and more effective.
```

Comment:
I think the background is fine. It isn't excessive or anything and doesn't distract from the point.

Reply:
I think the first paragraph is good—it gives people the links if they want to learn more. But the second paragraph has a little too much information and could be confusing. I think it could be simplified to "Python's iterator functionality can be complex and a bit difficult to use. Coconut simplifies many of these features, making using iterators easier and more effective."